.github/workflows/go.yml

```
@@ -51,6 +51,7 @@ jobs:
 
       - name: Run Tests
         run: go test -v ./...
+        run: go test -v ./... -tags=unsafe
```

actionlint (1.7.12)

[error] key "run" is duplicated in element of "steps" section. previously defined at line:53,col:9

(syntax-check)